graphics/DDA_Algorithm.c

```diff
@@ -0,0 +1,46 @@
+// DDA line drawing algorithm
+
+#include <graphics.h>
+#include <stdio.h>
+#include <conio.h>
+#include <math.h>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>     /// for IO operations
#include <assert.h>    /// for assert
```